src/synapso_cli/commands/cortex/commands.py

```
@@ -18,7 +19,17 @@ def create(
         ..., help="The name of the cortex to create", metavar="CORTEX_NAME"
     ),
 ):
-    cortex_id = cmd_create_cortex(folder_location, cortex_name)
+    rest_client = get_rest_client()
+    try:
+        response = rest_client.create_cortex(folder_location, cortex_name)
+    except SynapsoRestClientError as e:
+        typer.echo(f"Synapso REST client error: {e}", err=True)
+        raise typer.Exit(1)
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)